apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -330,6 +330,9 @@ function initializeBlocklyWrapper(blocklyInstance) {
     registerCustomProcedureBlocks() {
       // Google Blockly only. Registers custom blocks for modal function editor.
     },
+    partitionBlocksByType() {
+      // TODO: Add comment
```

Comment:
@molly-moen What do you think is the most useful way to phrase this comment?

Reply:
I'd probably do something like the comment on `registerCustomProcedureBlocks` that this function only applies to google blockly.